src/patternfly/components/VerticalNav/vertical-nav-link.hbs

```diff
@@ -0,0 +1,10 @@
+<a href="{{url}}" class="pf-c-vertical-nav__link {{#if current}}pf-m-active {{/if}}{{ pf-c-vertical-nav--modifier }}" role="link" {{#if current}} aria-current="true" {{/if}}>
```

Comment:
If this menu item has a submenu, then the `<a>` element should include `aria-expanded` set to `true` when expanded and `false` when collapsed.